cwl/src/main/scala/cwl/CommandLineTool.scala

```diff
@@ -86,6 +86,11 @@ case class CommandLineTool private(
         RequiredInputDefinition(FullyQualifiedName(cip.id).id, tpe)
       }.toList
 
+    def stringOrExpressionToString(soe: Option[StringOrExpression]): Option[String] = soe match {
+      case Some(StringOrExpression.String(str)) => Some(str)
```

Comment:
very small nit but if `match` were `flatMap` you could get rid of the `Some` in the case statement